docs/guide/docs/manage/configuration.md

```diff
@@ -0,0 +1,123 @@
+
+To get more information about each individual options, check out the [comments on the configuration schema](https://github.com/botpress/botpress/blob/master/src/bp/core/config/botpress.config.ts)
+
+## HTTP Server Configuration
+
+By default, Botpress will start an HTTP server on localhost, listening to port 3000. If the configured port is already in use, it will pick the next available one. You can change these by editing `httpServer.host` and `httpServer.port`.
+
+### Exposing your bot on the internet
+
+When you are ready to expose your bot externally, you will need to change some of the server settings. The server doesn't support HTTPS connections, so you will need to set up a reverse proxy in front of it (for example: NGINX).
+
+This means that your server will still listen for connections on port 3000, but your reverse proxy will answer for queries on port 80. It's also the reverse proxy that will handle secure connections if you want to access your bot using `https`
+
+At this point, Botpress doesn't know how to access the bot from the web. You will need to edit the configuration of `httpServer.externalUrl`. Set the configuration variable to the complete host name, for example `https://bot.botpress.com`
+
+## Logs Configuration
+
+Logs are very useful to debug and understand what happens when the bot doesn't behave as expected.
+
+When you start Botpress from the binary (or using the Docker image), the bot is in `debug` mode. This means that a lot of information will be displayed in the console to understand what happens.
+
+There are 4 different levels of logs:
+
+- Debug: display very detailed information about the bot operations
+- Info: gives general information or "good to know" stuff
+- Warn: means that something didn't go as expected, but the bot was able to recover
+- Error: there was an error that should be addressed
+
+By default, you will see `debug` with all other log levels in the console, and `errors` will be saved in the database (useful to keep track of them).
+When you start Botpress in `production` mode, `debug` logs will be disabled for better performances.
+
+It is also possible to send log output to a file in a specific folder. Check below for the required configuration
+
+### How to save logs on the file system
+
+Edit your `botpress.config.json` file and change your file to match the following:
+
+```js
+{
+  ...
+  "logs": {
+    ...
+    "fileOutput": {
+      "enabled": true,
+      "folder": "./", // Change this to any folder, by default it will be in the same folder as the executable
+      "maxFileSize": 10000 // By default, the maximum file size will be kept under 10mb
+    }
+  },
+}
+```
+
+## Enable or disable modules
+
+When you start Botpress for the first time, the most popular modules included with the binary will be added to your `botpress.config.json` file. If you want to disable or enable modules, you need to edit the `modules` option.
```

Comment:
Can also be toggled from the admin UI. I think this is documented somewhere else, juste make sure it doesn't confuse users to have that documented in 2 different locations

Reply:
I got this question a couple of times. From my understanding. 

- If you create a custom module. You need to [enable](https://botpress.com/docs/main/module#enabling-or-disabling-modules) it in the `package.json`. In other to see it in the Admin Panel. 
- Once the custom module is enabled. You can use the toggle knob (Module Panel) to enable the module inside botpress. 

This is correct 🤔 ?